### PR TITLE
`impl_tanssi_pallets_config!` macro for container chains

### PR DESCRIPTION
**Pull request by nanocryk**
Provide a new `impl_tanssi_pallets_config!` macro that implements `Config` from `pallet_author_inherent`, `pallet_timestamp` and `pallet_cc_authorities_noting` with the proper parameters to work with Tanssi.

Customizable parameters are set by impl `impl_tanssi_pallets_config::Config`.

The macro can't check at compile time that the pallets have been listed in the `construct_runtime!` macro, however it will generate a test for that.

### CODE REVIEW

**Review by girazoki — APPROVED** [review]
Love this! Maybe we should put the impl-tanssi-pallets-config in dancekit though, what do you think @nanocryk? Ideally dancekit should contain the tools to implement a runtime. But we can do it in a separate PR

File: container-chains/templates/frontier/runtime/src/lib.rs [hunk 1]
```diff
@@ -917,6 +889,15 @@ impl pallet_tx_pause::Config for Runtime {
     type WeightInfo = pallet_tx_pause::weights::SubstrateWeight<Runtime>;
 }
 
+impl tp_impl_tanssi_pallets_config::Config for Runtime {
+    const SLOT_DURATION: u64 = SLOT_DURATION;
```
[the remark this answers is not in the document]
Comment (girazoki) [reply container-chains/templates/frontier/runtime/src/lib.rs:893]: I think if they missconfigure it, they wont produce blocks. So yeah maybe its better if we set it ourselves. @nanocryk can you do that?

[the remark this answers is not in the document]
Comment (girazoki) [reply container-chains/templates/frontier/runtime/src/lib.rs:893]: it is true that maybe it provides a better protection against evm, so I will change that

[the remark this answers is not in the document]
Comment (tmpolaczyk) [reply container-chains/templates/frontier/runtime/src/lib.rs:893]: Maybe let's keep this until we change it to 6 seconds, the transition will be easier if this is configurable

[the remark this answers is not in the document]
Comment (girazoki) [reply container-chains/templates/frontier/runtime/src/lib.rs:893]: "easier"

[the remark this answers is not in the document]
Comment (tmpolaczyk) [reply container-chains/templates/frontier/runtime/src/lib.rs:893]: :sob:

**Comment by girazoki** [comment]
@nanocryk can you merge master back? I think we can merge this after.

**Review by tmpolaczyk — APPROVED** [review]

**Pull request merged**